src/main/java/core/basesyntax/model/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started.");
```

Comment:
The message in the `doWork` method should be more informative and indicate that it is an `Excavator` that is working. Consider including the type of machine in the message, for example: 'Excavator is working.'